src/shared/components/common/wine-progress/index.tsx

```
@@ -0,0 +1,49 @@
+  avgRatings: {
+    1: 3,
+    2: 8,
+    3: 11,
+    4: 14,
+    5: 19,
+  },
+  reviewCount: 55,
+};
+
+// 상위 컴포넌트에서 wineId를 전달받아서 커스텀 훅으로 api 통신을 통해 데이터 가져오기
+const WineProgress = ({ wineId }: WineProgressProps) => {
```

ESLint

[error] 'wineId' is defined but never used.

(@typescript-eslint/no-unused-vars)